app/views/user/_exercises_table.erb

```diff
@@ -18,7 +18,7 @@
       <th>Language</th>
       <th>Exercise</th>
       <th>Iterations</th>
-      <th data-sorted="true" data-sorted-direction="descending">Latest Iteration</th>
+      <th data-sorted="true" data-sorted-direction="ascending">Latest Iteration</th>
```

Comment:
I believe the change to the Ruby list generation would be a better solution, but I haven't learned Ruby yet, so I proposed a solution that will work for the desired output result.  If someone wants to change the Ruby, please take this.  Otherwise, the change to the HTML should result in the desired display order.